jwst/photom/photom.py

```diff
@@ -990,8 +991,8 @@ def apply_photom(self, photom_fname, area_fname):
 
         # Load the pixel area reference file, if it exists, and attach the
         # reference data to the science model
-        if area_fname != 'N/A':
-            self.save_area_info(ftab, area_fname)
+        #if area_fname != 'N/A':
+        self.save_area_info(ftab, area_fname)
```

Comment:
Always call the ``save_area_info`` function and handle the absence of an AREA reference file within the function itself.